privacy/index.md

```diff
@@ -21,7 +21,7 @@ When you sign up for a Basecamp product, we typically ask for identifying inform
 
 ### Billing information
 
-When you pay for a Basecamp product, we ask for your credit card and billing address. That's so we can charge you for service, calculate taxes due, and send you invoices. Your credit card is passed directly to our payment processor and doesn't ever go through our servers. We store a record of the payment transaction, including the last 4 digits of the credit card number and as-of billing address, for account history, invoicing, and billing support. We store your billing address to calculate any sales tax due in the United States, to detect fraudulent credit card transactions, and to print on your invoices.
+When you pay for a Basecamp product, we ask for your credit card and billing address. That’s so we can charge you for service, calculate taxes due, and send you invoices. Your credit card is passed directly to our payment processor and doesn't ever go through our servers. We store a record of the payment transaction, including the last 4 digits of the credit card number and as-of billing address, for account history, invoicing, and billing support. We store your billing address to calculate any sales tax due in the United States, to detect fraudulent credit card transactions, and to print on your invoices.
```

Comment:
Billing address is now also used for calculating sales taxes due in Europe for HEY for personal use.

Reply:
Good catch! I’ll address this in a separate PR for easier finding. Thanks.